guide/ops/cli/index.md

```diff
@@ -6,7 +6,25 @@ children:
 - cli-usage-guide.md
 ---
 
-{% include list-children.html %}
 
-**NOTE:** These documents are for using the Brooklyn Client CLI to access a running Brooklyn Server.  For 
+**NOTE:** These documents are for using the Brooklyn Client CLI tool to access a running Brooklyn Server.  For
 information on starting on a Brooklyn Server, refer to [Server CLI Reference](../server-cli-reference.html).
+
+## Obtaining the CLI tool
+
+The CLI tool, `br`,  is included in the Apache Brooklyn distribution, in bin/brooklyn-client-cli-0.9.0-SNAPSHOT.
+The tool is an executable binary (written in Go), and is distributed in a number of builds for a variety of platforms
+and architectures, each build in its own subdirectory:
+
+    darwin.386    freebsd.386    linux.386    netbsd.386     openbsd.386    windows.386
+    darwin.amd64  freebsd.amd64  linux.amd64  netbsd.amd64   openbsd.amd64  windows.amd64
```

Comment:
i think supporting windows CLI is a priority.  suggest fixing it rather than changing the docs.

will you also exclude the 6 bsd items and the 32-bit darwin item?  assuming yes i'll update the other PR's and merge.